src/lib/ui/thermostat/AccordionRow.tsx

```diff
@@ -0,0 +1,44 @@
+import { type PropsWithChildren, useState } from 'react'
+
+import { ChevronWideIcon } from 'lib/icons/ChevronWide.js'
+
+interface AccordionRowProps {
+  title: string
+  triggerRightContent?: JSX.Element
```

Comment:
This reads like a verb, but it needs to be a noun. Having a hard time naming it though. Maybe

```
  expanderContent?: JSX.Element
```

Or 

```
  rightCollapsedContent?: JSX.Element
```

Reply:
Ahh, nice catch. I'll go with the latter I think.